code/modules/admin/admin_verbs.dm

```diff
@@ -937,8 +937,9 @@ var/list/admin_verbs_mod = list(
 		var/obj/item/weapon/reagent_containers/food/drinks/golden_cup/C = new(get_turf(winner))
 		C.name = name
 		C.desc = desc
-		winner.put_in_hands(C)
-		winner.update_icons()
+		if(iscarbon(winner) && (winner.stat == CONSCIOUS))
+			winner.put_in_hands(C)
+			winner.update_icons()

```

Comment:
This isn't necessary.
The default put_in_hands() behaviour already updates icons.